packages/components/pie-button/src/button.scss

```diff
@@ -45,13 +44,59 @@
     user-select: none;
 
     &:focus-visible {
-        // TODO - Need to update to use the latest focus styling
-        outline: 2px solid var(--btn-focus);
+        /*
+        TODO: This focus styling should be extracted out into a shared mixin to share among other components
+        Applying both shadows in a single declaration gives anti-aliasing artefacts.
+        box-shadow: 0 0 0 2px dt.$color-focus-inner, 0 0 0 4px dt.$color-focus-outer;
+
+        To prevent this, they should be separated so they don't bleed into each other,
+        i.e., one shadow on the element and another on a pseudo element like :after
```

Comment:
Recommend always referencing a JIRA ticket to todo's - so you have a two-way means to resolve the issue and know whether it's still relevant or not.

Reply:
We're just opening a conversation about documenting conventions like this – this sounds like a good one. I'll brig it up there.

We'll be creating some tickets around the CSS tidyup in the next couple of weeks, so once that's done I'll add the ticket number to this TODO 👍